app/controllers/simplified-account/settings/api-keys/api-keys.controller.test.js

```diff
@@ -0,0 +1,49 @@
+const ControllerTestBuilder = require('@test/test-helpers/simplified-account/controllers/ControllerTestBuilder.class')
+const sinon = require('sinon')
+const { expect } = require('chai')
+const ACCOUNT_TYPE = 'live'
+const SERVICE_ID = 'service-id-123abc'
+
+const mockResponse = sinon.spy()
+const tokens = [{ description: 'my token', created_by: 'system generated', issued_date: '12 Dec 2024' }]
+const mockPublicAuthService = {
+  getActiveTokensForAccount: sinon.stub().resolves({
+    tokens
+  })
+}
+
+const {
+  req,
+  res,
+  call
+} = new ControllerTestBuilder('@controllers/simplified-account/settings/api-keys/api-keys.controller')
+  .withServiceExternalId(SERVICE_ID)
+  .withAccountType(ACCOUNT_TYPE)
+  .withStubs({
+    '@utils/response': { response: mockResponse },
+    '@services/clients/public-auth.client': mockPublicAuthService
+  })
+  .build()
+
+describe('Controller: settings/api-keys', () => {
+  describe('get', () => {
+    before(() => {
+      call('get')
+    })
+
+    it('should call the response method', () => {
+      expect(mockResponse.called).to.be.true // eslint-disable-line
```

Comment:
sinon-chai assertions are quite nice for this, you can do `expect(mockResponse).to.have.been.calledOnce`